scripts/molar_concentration.py

```diff
@@ -25,12 +25,20 @@ def apply_calculations(lims, artifacts, conc_udf, size_udf, unit_udf, epp_logger
                      "Concentration: {1}, Size: {2}, ").format(artifact.id,
                                                         artifact.udf[conc_udf],
                                                         artifact.udf[size_udf]))
-        factor = 1e6 / (328.3 * 2 * artifact.udf[size_udf])
-        artifact.udf[conc_udf] = artifact.udf[conc_udf] * factor
+        artifact.udf[conc_udf] = calculate_fmol(artifact.udf[conc_udf], artifact.udf[size_udf])
         artifact.udf[unit_udf] = 'nM'
         artifact.put()
         logging.info('Updated {0} to {1}.'.format(conc_udf,
                                                  artifact.udf[conc_udf]))
+
+def calculate_fmol(amount_ng, size_bp):
```

Comment:
Hi, just a minor comment: could it be better to rename `amount_ng` into `conc_ng_ul`? The input is from the conc_udf, and naming it as "amount" might be missleading.

Reply:
[f270328](https://github.com/SciLifeLab/scilifelab_epps/pull/242/commits/f2703287d21b66eb4d3a5bd6adc08391290d43cd)